test/e2e/helpers/perf-utils.ts

```diff
@@ -0,0 +1,39 @@
+import { Metrics } from 'puppeteer';
+
+/**
+ * Copyright 2020 Bonitasoft S.A.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+export interface PerformanceMetric {
+  run: number;
+  TaskDuration: number;
+  ScriptDuration: number;
+  RecalcStyleDuration: number;
```

Comment:
:question: can we rename the `recalc` part into something clearer?

Reply:
same as above, puppeteer object has this props, wanted to stay coherent
@see https://pptr.dev/#?product=Puppeteer&show=api-pagemetrics